.github/instructions/repository-structure.instructions.md

```diff
@@ -0,0 +1,91 @@
+│
+├── Directory.Build.props         # Root build properties (all projects)
+├── Directory.Packages.props      # Central Package Management
+├── Tests.props                   # Shared test configuration
+└── .editorconfig                 # Root code style rules
+```
+
+## Key Architecture Layers
+
+| Layer | Purpose | Example Projects | Dependencies Flow |
+|-------|---------|-----------------|-------------------|
+| Domain | Business logic and entities | Lewee.Domain, Pizzeria.Store.Domain | No dependencies on other layers |
+| Application | Use cases and orchestration | Lewee.Application, Pizzeria.Store.Application | Depends on Domain |
+| Infrastructure | External concerns (DB, API) | Lewee.Infrastructure.*, Pizzeria.Store.Data | Depends on Application and Domain |
+| Presentation | User interface | Pizzeria.Store.Api, Lewee.Blazor | Depends on all layers |
+
+## Configuration Files
+
+| File | Purpose | When to Edit |
+|------|---------|-------------|
+| `lewee.sln` | Solution with all projects | Adding/removing projects |
+| `Directory.Build.props` | Global MSBuild properties (targets .NET 10.0) | Changing global build settings |
+| `src/Directory.Build.props` | Framework package properties | Changing framework-specific settings |
+| `tests/Directory.Build.props` | Test project properties | Changing test-specific settings |
+| `sample/Directory.Build.props` | Sample application properties | Changing sample-specific settings |
+| `sample-tests/Directory.Build.props` | Sample test properties | Changing sample test-specific settings |
+| `Directory.Packages.props` | Central Package Management (CPM) | Adding/updating NuGet packages |
+| `Tests.props` | Shared test configuration | Changing test framework or packages |
+| `.editorconfig` | Global code style rules | Adjusting global code formatting |
+| `tests/.editorconfig` | Test-specific analyzer rules | Adjusting test-specific rules |
+| `sample/.editorconfig` | Sample-specific analyzer rules | Adjusting sample-specific rules |
+| `sample-tests/.editorconfig` | Sample test-specific analyzer rules | Adjusting sample test-specific rules |
+| `sample/Pizzeria.AppHost/Program.cs` | Aspire orchestration | Configuring services |
+| `.github/workflows/ci.yml` | CI/CD pipeline | Modifying build/test process |
+
+## Common Development Tasks
+
+| Task | Command | Use Case |
+|------|---------|----------|
+| Clean artifacts | `dotnet clean lewee.sln` | Remove build outputs |
+| Full rebuild | `dotnet build lewee.sln --configuration Release --no-incremental` | After major changes |
+| Run specific tests | `dotnet test tests/Lewee.Domain.Tests.Unit/` | Test single project |
+| Run sample app | `dotnet run --project sample/Pizzeria.AppHost/` | Manual testing |
+| Check outdated packages | `dotnet list package --outdated` | Dependency updates |
+| Format code | `dotnet format lewee.sln` | Fix style issues |
+| Create packages | `dotnet pack lewee.sln --configuration Release --nologo` | Prepare for release |
```

Comment:
The solution file is not required as an arg to the `dotnet` commands as there's only one solution file in the root directory of the repository